src/devtools/useAtomsSnapshot.ts

```diff
@@ -1,6 +1,6 @@
 import { useContext, useMemo } from 'react'
 import { SECRET_INTERNAL_getStoreContext as getStoreContext } from 'jotai'
-import type { AnyAtom } from '../core/types'
+import type { AnyAtom, Scope } from '../core/types'
```

Comment:
```suggestion
import type { AnyAtom } from '../core/types'
```
It's not used, right?